lib/rspec/core/formatters/exception_formatter.rb

```diff
@@ -0,0 +1,136 @@
+module RSpec
+  module Core
+    module Formatters
+      # @private
+      class ExceptionFormatter
```

Comment:
Is `ExceptionFormatter` a good name for this? It's not a "Formatter" in RSpec's sense (even though it does do formatting), we use the term printer elsewhere for this sort of thing.